### PR TITLE
Fix genesis indexing txs order

### PR DESCRIPTION
**Pull request by ssd04**
Set minting transactions from genesis to be set in the first miniblock, handled by metachain.
 - handle mint miniblock separately

### CODE REVIEW

File: genesis/parsing/accountsParser.go [hunk 1]
```diff
@@ -362,9 +362,35 @@ func createMiniBlocks(shardIDs []uint32, blockType block.Type) []*block.MiniBloc
 	return miniBlocks
 }
 
+func (ap *accountsParser) setMintingTxsPoolAndMiniBlock(
+	txs []coreData.TransactionHandler,
+	txsPoolPerShard map[uint32]*indexer.Pool,
+) (*block.MiniBlock, error) {
+	txHashes := make([][]byte, 0)
```
Comment (miiu96) [inline genesis/parsing/accountsParser.go:369]: `txHashes := make([][]byte, 0, len(txs))`

File: genesis/parsing/accountsParser.go [hunk 2]
```diff
@@ -362,9 +362,35 @@ func createMiniBlocks(shardIDs []uint32, blockType block.Type) []*block.MiniBloc
 	return miniBlocks
 }
 
+func (ap *accountsParser) setMintingTxsPoolAndMiniBlock(
```
Comment (miiu96) [inline genesis/parsing/accountsParser.go:365]: maybe you can change the name of the function 
`putMintingTxsInPoolAndCreateMiniblock`

File: genesis/parsing/accountsParser.go (hunk 1 above)
Comment (ssd04) [reply genesis/parsing/accountsParser.go]: done

File: genesis/parsing/accountsParser.go (hunk 2 above)
Comment (ssd04) [reply genesis/parsing/accountsParser.go]: :+1:

**Review by gabi-vuls — APPROVED** [review]
System test passed. Errors and warnings are safe .

@@                            Log scanner                                  @@
##                            fix-genesis-indexing-txs-order                                       ##
================================================================================
- Known Warnings              22
- New Warnings                5
- Known Errors                0
- New Errors                  2
- Panics                      0
================================================================================

**Review by miiu96 — APPROVED** [review]

**Pull request merged**